### PR TITLE
Succeed in 441

### PR DESCRIPTION
**Pull request by dylanfeehan**
added a folder in guides/academics/succeed-in-x, where we write guides for how to succeed in certain courses. I started it off with a first draft of how to succeed in 441.

### CODE REVIEW

**Comment by EParmar18** [comment]
Think you have a small spelling error "This course will focus heavily on the **itegers** (which are discrete by nature), whereas a continuous field like calculus focuses on real numbers"

**Review by Zmwang622 — CHANGES_REQUESTED** [review]
going to request changes for now, bc i have some comments that ill write later

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+You don't have to follow all of these guidelines to pass the course, but these are my recommendations if you want to walk away with as much knowledge as possible.
+
+### **High level ideas**
+
+- Have a problem solving mindset, and practice proofing.
+- Go to office hours.
+- Engage with the course material before, during, and after lecture.
+- Let yourself get stuck.. then get unstuck. 
+- If you're *really* stuck, don't look up the answer (especially on a homework, thats cheating). Often times is helpful to step away from a problem or even sleep on it.
```
Comment (EParmar18) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:41]: "Often times **is** helpful to step away from a problem or even sleep on it." - grammar thing, probably change to it's or it is

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### **Homeworks**
+
+- Start the homeworks as early as possible. You don't want to be stuck having to write 5+ proofs with 3 hours until the submission deadline, trust me. If you get stuck on a problem, move on and finish as many as you can. Then, go to office hours and address the problems that you are stuck on. Also, do the bonus problems.
+- The homeworks will start out pretty easy, but they will ramp up in difficulty very quickly. Once proofs are introduced, homeworks will probably take about twice the amount of that they did before.
+
+### **Studying for exams**
+
+- As with most CS courses, avoid rote memorizaiton. Computer science is very a conceptual field and memorization plays a miniscule role. What's important is understanding the systems / patterns, conceptualization, and problem solving, all of which are skills that can be strengthented with practice. The odd-numbered practice problems at the end of every chapter are very helpful and their answers can be found at the end of the book.
```
Comment (EParmar18) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:58]: "all of which are skills that can be **strengthented** with practice" - spelled strengthened wrong here

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- The homeworks will start out pretty easy, but they will ramp up in difficulty very quickly. Once proofs are introduced, homeworks will probably take about twice the amount of that they did before.
+
+### **Studying for exams**
+
+- As with most CS courses, avoid rote memorizaiton. Computer science is very a conceptual field and memorization plays a miniscule role. What's important is understanding the systems / patterns, conceptualization, and problem solving, all of which are skills that can be strengthented with practice. The odd-numbered practice problems at the end of every chapter are very helpful and their answers can be found at the end of the book. 
+
+## Conclusion:
+
+You can learn a *massive* amount of information from this course. You will have better problem solving skills, you'll be better prepared to understand theoretical topics in CS, and your logical thinking will be much more sound. I will admit this was a very challenging course. But if you spend enough time engaging with the material, you can and will get very comfortabile with it. Remember, if a course isn't challenging, it's probalby not increasing the value of your education.
```
Comment (EParmar18) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:62]: "I will admit this was a very challenging course. But if you spend enough time engaging with the material, you can and will get very **comfortabile** with it" - spelled comfortable wrong and a grammatical change, probably change it to
  "I will admit this is a challenging course, but if you spend enough time engaging with the material you will succeed!"

Comment (EParmar18) [reply src/guides/academics/succeed-in-course-x/succeed-in-441.md:62]: Also spelled ** probably ** wrong

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- The homeworks will start out pretty easy, but they will ramp up in difficulty very quickly. Once proofs are introduced, homeworks will probably take about twice the amount of that they did before.
+
+### **Studying for exams**
+
+- As with most CS courses, avoid rote memorizaiton. Computer science is very a conceptual field and memorization plays a miniscule role. What's important is understanding the systems / patterns, conceptualization, and problem solving, all of which are skills that can be strengthened with practice. The odd-numbered practice problems at the end of every chapter are very helpful and their answers can be found at the end of the book. 
+
+## Conclusion:
+
+You can learn a *massive* amount of information from this course. You will have better problem solving skills, you'll be better prepared to understand theoretical topics in CS, and your logical thinking will be much more sound. I will admit this is a challenging course, but if you spend enough time engaging with the material you will succeed! Remember, if a course isn't challenging, it's probalby not increasing the value of your education.
```
Comment (EParmar18) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:63]: "it's **probalby** not increasing the value of your education" - probably still spelled wrong

**Comment by dylanfeehan** [comment]
> going to request changes for now, bc i have some comments that ill write later

yo dude can u just let me in

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  - "441"
+---
+
+Preface: *If you want to do well in any course, the bare minimum is attending all lectures and doing all homework assignments.*
+
+## What is *Discrete Structures for Computer Science?*  
+
+CS 0441 Discrete Structures for Computer Science is a mathematics course which focuses on the study of mathematical structures that are discrete rather than continuous. This means that the objects within the mathematical structures that you'll learn about are distincly separated values. This course will focus heavily on the integers (which are discrete by nature), whereas a continuous field like calculus focuses on real numbers. Lets start with a birds-eye view of the main topics covered in 441:
+- Logic and Proofs
+  - This is the theoretical foundation of the course which teaches you how to derive additional information from some given information. Thoroughly understanding the proofing techniques is incredibly important because you will be using these techniques throughout the rest of the course.
```
Comment (Zmwang622) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:13]: I think the content you wrote here is great, but I recommend breaking it down into more bullet points for easier reading.

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+### **Preparation**
+
+- To specify, this section will be about the preparation you can do before the *semester*. Walking into the first day of lecture with a solid understanding of the course material is a fantastic learning advantage and confidence booster. Before the semester starts, I recommend learning a little bit about discrete math on your own. Regarding depth at which you should read / learn, I recommend going an inch deep and a mile wide. Instead of learning only the *first* course topic in its entirety, learn a little bit about all of the topics. Learning everything about logic and proofs before the semester will just make the first month of class boring, and the learning benefits will be over as soon as the next topic beings (which means your workload will increase quite a bit). If you learn a little bit about every topic, you'll have the learning advantage throughout the entire semester, and your workload will stay more consistent. That being said, if you find yourself excited about a particular topic and you wish to read deeper, do it!!!!
```
Comment (Zmwang622) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:58]: once again we probably shouldn't have a bullet here. Either break it up into multiple bullets or just have it be a paragraph (which I think is better for this specific section)

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+### **The textbook**
+
+- *Discrete Mathematics and its Applications* by Kenneth Rosen is a fantastic textbook. The concepts are explained well and there are in-text exercises to get you ready for the end-of-chapter problems. These in text exercises are notated by **EXAMPLE X** in the left margin, and they consist of a small problem followed by a solution. **Try solving the exercises by hand before looking at the solution.** This is a fantastic way to engage with the material before class, even if you only do a few problems. At the end of each chapter is around 50 practice problems, and the solutions to the odd-numbered problems can be found at the end of the book. A free pdf version of the textbook can be found easily. I'm not going to provide the link because that's probably crime, but here's something: https://bit.ly/3wB7BZi
```
Comment (Zmwang622) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:66]: maybe link to the textbook?

**Review by Zmwang622 — APPROVED** [review]
For the time being this is good, we'll probably turn this guide into a separate course guide when we have more of these sort of stuff written out. 

Please don't merge until you address my other comments though. Main issue is that a list with just one bullet point and long ass paragraph is not a great UX choice (bullets are supposed to drill topics down into atomic snippets).

 For the headers under `How to succeed in 441` (ex: Preparation, Readings, The textbook, Homeworks, Studying), consider just making those paragraphs.  Also consider breaking those large paragraphs into smaller ones. It's just not easy to read a large chunk of text.

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+- The homeworks will start out relatively easy, but they will ramp up in difficulty very quickly. Once proofs are introduced, homeworks will probably take about twice the amount of that they did before.
+
+### **Studying for exams**
+
+- As with most CS courses, avoid rote memorization. Computer science is very a conceptual field and memorization plays a miniscule role. What's important is understanding the systems / patterns, conceptualization, and problem solving, all of which are skills that can be strengthened with practice. The odd-numbered practice problems at the end of every chapter are very helpful and their answers can be found at the end of the book.
+
+## Conclusion:
+
+You can learn a *massive* amount of information from this course. You will have better problem solving skills, you'll be better prepared to understand theoretical topics in CS, and your logical thinking will be much more sound. I will admit this is a challenging course, but if you spend enough time engaging with the material you will succeed! Remember, if a course isn't challenging, it's probably not increasing the value of your education.
```
Comment (Zmwang622) [inline src/guides/academics/succeed-in-course-x/succeed-in-441.md:81]: also you use the word I alot but we don't know who I is. Either add a author tag in the front matter ([example of front matter](https://github.com/PittCSWiki/pittcswiki/blob/ef03b24dd6d58b5e0bce70ead86028d7658fbfff/src/guides/academics/prospective/coe-vs-cs.md)) or change the I's to we's.

File: src/guides/academics/succeed-in-course-x/succeed-in-441.md (hunk 5 above)
Comment (dylanfeehan) [reply src/guides/academics/succeed-in-course-x/succeed-in-441.md:13]: which content? the topics covered or something else?

**Pull request merged**